Library/Formula/fpc.rb

```diff
@@ -28,11 +28,15 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to set up unit search paths
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{etc}/fpc.cfg"
   end
 
   test do
     hello = <<-EOS.undent
       program Hello;
+      uses GL;
```

Comment:
it's to make sure paths are correctly set up, an unused unit will be simply ignored

```
 $ brew test --verbose fpc
Testing fpc
==> Using the sandbox
/usr/bin/sandbox-exec -f /tmp/homebrew20160218-35257-4zttbi.sb /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby -W0 -I /usr/local/Library/Homebrew -- /usr/local/Library/Homebrew/test.rb /usr/local/Library/Formula/fpc.rb --verbose
==> /usr/local/Cellar/fpc/3.0.0/bin/fpc hello.pas
Free Pascal Compiler version 3.0.0 [2016/02/15] for x86_64
Copyright (c) 1993-2015 by Florian Klaempfl and others
Target OS: Darwin for x86_64
Compiling hello.pas
Assembling (pipe) hello.s
Linking hello
5 lines compiled, 0.1 sec
```